routes/admin.route.js

```diff
@@ -151,6 +151,41 @@ router.put(
 
 /**
  * @swagger
+ * /admin/disable:
+ *   put:
+ *     tags: 
+ *       - User
+ *     summary: Disable a Specified User
+ *     description: Used to disable a user account conditional on admin privilages 
+ *     responses:
+ *       '200':
+ *          description: A successful request, user is disabled
+ *       '409':
+ *          description: The Specified User is already disabled
+ *       '500': 
+ *          description: Internal server error
+ */
+
+router.put(
+  "/disable",
+  [
+    body("username")
```

Comment:
We need to change this verification to the proper name of the field you are checking for. Update the message as well to the proper message.